insights/parsers/tomcat_xml.py

```diff
@@ -0,0 +1,86 @@
+"""
+tomcat_xml - XML files for Tomcat
+=================================
+
+Classes to parse Tomcat XML configuration files.
+
+.. note::
+    The tomcat XML files are gotten from the directory specified in the java
+    commands
+
+"""
+import xml.etree.ElementTree as ET
+from .. import Parser, parser, LegacyItemAccess
+from insights.specs import tomcat_web_xml
+
+
+@parser(tomcat_web_xml)
```

Comment:
Hey @xiangce , as we discussed yesterday, we may need to create a base class for XMLParser, like https://github.com/RedHatInsights/insights-core/pull/855. I think we'd better re-update this PR when the base class is merged.

Reply:
Sure, Looking forward to the new XMLParser you created.